docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -109,29 +110,27 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
-$number-of-card-columns-gt-sm: 5;
-$number-of-card-columns-sm-max: 3;
-$number-of-card-columns-xs-max: 1;
 
 .components {
   display: grid;
   letter-spacing: 3px;
-  grid-template-columns: repeat($number-of-card-columns-gt-sm, 1fr);
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
```

Comment:
I'm pretty sure the `1fr` part should be removed, cards should always have the same width
As the viewport increases, you should add more space to the left and right of the grid instead of distorting the cards